ci/jenkins/test.sh

```diff
@@ -621,6 +627,9 @@ function deliver_antrea {
             scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" flow-aggregator.tar jenkins@[${IP}]:${DEFAULT_WORKDIR}/flow-aggregator.tar
             ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" -n jenkins@${IP} "${CLEAN_STALE_IMAGES}; docker load -i ${DEFAULT_WORKDIR}/antrea-ubuntu.tar; docker load -i ${DEFAULT_WORKDIR}/flow-aggregator.tar" || true
         done
+    elif [[ $TESTBED_TYPE == "kind" ]]; then
+            kind load docker-image antrea/antrea-ubuntu:latest --name ${KIND_CLUSTER}
+            kind load docker-image antrea/flow-aggregator:latest --name ${KIND_CLUSTER}
```

Comment:
not sure it is worth re-organizing the code, but for the Kind case, the `docker save` commands above are not needed

Reply:
yes , but I think for other testbed_type we need that and if we remove those cmds then maybe it will affect other testbed_type, actually we still have some other improvements or refactoring to do in this task you can check our comments on the PR, currently I have added  6 vms to handle parallel runs but for the theory changes  will have a follow-up PR. 
And in that PR we will cover two cases: 
1. there could be chance that Job is aborted then the kind cluster becomes garbage so we need to handle it.
2.  Refactor Antrea build script and kind test script to allow multiple kind tests run on single VM.

could you please correct me?

Reply:
The `docker save` command should not take too long, so it's not worth making the code more complex to avoid running the command for the kind testbed type